session/bootstrap.go

```diff
@@ -2773,10 +2777,16 @@ func upgradeToVer171(s Session, ver int64) {
 	if ver >= version171 {
 		return
 	}
+	mustExecute(s, "ALTER TABLE mysql.tidb_runaway_queries CHANGE COLUMN `tidb_server` `tidb_server` varchar(512)")
```

Comment:
Due to TestUpgradeVersionForResumeJob, I try to pass it by add two upgrade functions. Maybe it's not a good method, I just want to pass test first.